qutip/core/superoperator.py

```diff
@@ -269,6 +269,7 @@ def operator_to_vector(op):
     return Qobj(stack_columns(op.data),
                 dims=[op.dims, [1]],
                 type='operator-ket',
+                superrep="super",
```

Comment:
Could we add direct unit tests (or asserts to existing tests) that check that:

1. `operator_to_vector` sets `superrep` correctly now.
2. `vector_to_operator` raises the expected error if an `operator-ket` with the wrong `superrep` is given.

Reply:
Separately, I'm wondering if `operator_to_vector` is missing some input checks? Can one really call it on any Qobj?

Reply:
I added input check. Since we support non-square super-operator, the check is no very strict. We may want to review this once we have better dimension handling.